diff --git a/vendor/magento/module-ui/view/base/web/js/form/element/file-uploader.js b/vendor/magento/module-ui/view/base/web/js/form/element/file-uploader.js
index 357571350a26..c6bf36f5cc86 100644
--- a/vendor/magento/module-ui/view/base/web/js/form/element/file-uploader.js
+++ b/vendor/magento/module-ui/view/base/web/js/form/element/file-uploader.js
@@ -16,7 +16,8 @@ define([
     'Magento_Ui/js/form/element/abstract',
     'mage/backend/notification',
     'mage/translate',
-    'jquery/file-uploader'
+    'jquery/file-uploader',
+    'mage/adminhtml/tools'
 ], function ($, _, utils, uiAlert, validator, Element, notification, $t) {
     'use strict';
